src/EnergyPlus/StandardRatings.cc

```diff
@@ -964,7 +964,8 @@ namespace StandardRatings {
         int spnum; // compressor speed number
 
         // Calculated and reported to the EIO file
-        static Real64 SEER(0.0);                       // Seasonal Energy Efficiency Ratio in SI [W/W]
+        static Real64 SEER(0.0);                       // Seasonal Energy Efficiency Ratio using user PLF curve in SI [W/W]
+        static Real64 SEER_Default(0.0);               // Seasonal Energy Efficiency Ratio using AHRI 210/240 PLF default curve & C_D in SI [W/W]
```

Comment:
I would be inclined to use `SEER_User` and `SEER_Default` (or `_Standard`) to keep this very clear for future developers, here and everywhere else.

Reply:
Renamed SEER and SEER_Default to SEER_User ad SEER_Standard, respectively, every where.